### DISCUSSION TITLE
وصل نشدن به درگاه

### DISCUSSION THREAD

**Discussion by sinath72** [Sina-Soltani/Parbad#197] [Q&A]
سلام
من برای درگاه ایدی پی مشکل دارم و طبق کد ها هم پیش رفتم ولی هیج صفحه ای بالا نمیاد
میشه یکم کمکم کنید

اینم کد ها

کد فراخوانی

> pay_on(10,"test");

کد اصلی

> 
private readonly IOnlinePayment _onlinePayment;
private Task pay_on(int cost, string description)
        {
            Parbad.Gateway.IdPay.IdPayGatewayAccount idPayGatewayAccount = new Parbad.Gateway.IdPay.IdPayGatewayAccount();
            idPayGatewayAccount.Api = "X-API KEY";
            idPayGatewayAccount.IsTestAccount = true; 
            var result =  _onlinePayment.RequestAsync(invoice =>
        {
            invoice
               .SetTrackingNumber(123)
               .SetAmount(cost)
               .SetCallbackUrl("https://streamershow.ir")
               .SetGateway("IdPay");
            System.Action<Parbad.InvoiceBuilder.IInvoiceBuilder> action;
            // OR
            //.UseParbadVirtual()
        });
            return result;
}

**Comment by Sina-Soltani** [comment]
درود

کدهایی که نوشتید کاملا اشتباهه
آموزش ها موجوده
لطفا طبق آموزش ها پیش برید
[آموزش](https://github.com/Sina-Soltani/Parbad/wiki)
[مثال‌ها](https://github.com/Sina-Soltani/Parbad.Samples)

**Comment by parvizfahimi** [comment]
سلام.متاسفانه برای 
 asp.net webform
.من نتونستم بفهمم.اگه میشه نمونه کد رو با توضیحات بیشتر بزارید.
مثلا برای 
idpay 
بعد از نصب پرباد چیکار کنیم.هر چی کد  میزنم یه جاش خطا میخوره.
مرسی

**Reply by Sina-Soltani** [reply]
درود

آموزش‌ها و نمونه پروژه ها موجوده
دانلود کنید و تست کنید
[آموزش](https://github.com/Sina-Soltani/Parbad/wiki)
[مثال‌ها](https://github.com/Sina-Soltani/Parbad.Samples)
